src/pages/event/eventDetail/eventDetail.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { FunctionComponent } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import { EventProps } from "../../../components/UI/ResourceEvent";
+import { events } from "../event";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import { ChevronLeft, ExternalLink, PlayCircle, Download } from "react-feather";
+import { Page } from "../../../components/Layout/Page";
+import { isFuture, format } from "date-fns";
+import { formatTime } from "../../../common/utils/dateTime";
+import { LinkButton } from "@govtechsg/tradetrust-ui-components";
+import ReactMarkdown from "react-markdown";
+
+export const EventPageDetail: FunctionComponent = () => {
+  const locationPath = useLocation();
+  const params: { slug: string } = useParams();
+  const detail: EventProps = events.find((eventDetail) => eventDetail.slug === params.slug);
+
+  const {
+    title,
+    thumbnail,
+    blurb,
+    link,
+    date,
+    timeStart,
+    timeEnd,
+    videoLink,
+    slides,
+    registerLink,
+    location,
+    downloadableMediaContent1,
+    downloadableMediaContent2,
+    downloadableMediaContent3,
+    eventDetails,
+  } = detail.attributes;
+  const downloadableContents = [downloadableMediaContent1, downloadableMediaContent2, downloadableMediaContent3];
+
+  return (
+    <>
+      <Helmet>
+        <meta
+          property="description"
+          content="Check out our events and browse through our latest news and official statements."
+        />
+        <meta
+          property="og:description"
+          content="Check out our events and browse through our latest news and official statements."
+        />
+        <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+        <meta property="og:url" content={`${window.location.origin}${locationPath.pathname}`} />
+        <title>TradeTrust - {title}</title>
+      </Helmet>
+      <Page title="Event">
```

Comment:
there's some misalignment causing "jump", may want to compare how news was implemented

![Screenshot 2021-12-22 at 10 41 38 AM](https://user-images.githubusercontent.com/4774314/147026283-cac64bdb-c2cc-43c2-a051-8b1880c4af4e.png)

Reply:
ok thanks for noticing it, updated